### PR TITLE
llvm 12.0.1

### PR DESCRIPTION
**Pull request by abpostelnicu**
- [x ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/llvm.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Llvm < Formula
   desc "Next-gen compiler infrastructure"
   homepage "https://llvm.org/"
-  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/llvm-project-12.0.0.src.tar.xz"
-  sha256 "9ed1688943a4402d7c904cc4515798cdb20080066efa010fe7e1f2551b423628"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.1/llvm-project-12.0.1.src.tar.xz"
+  sha256 "129cb25cd13677aad951ce5c2deb0fe4afc1e9d98950f53b51bdcfb5a73afa0e"
   # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
   license "Apache-2.0" => { with: "LLVM-exception" }
   revision 1
```
Comment (nandahkrishna) [inline Formula/llvm.rb:8]:
```suggestion
```

Comment (nandahkrishna) [reply Formula/llvm.rb:8]: Note: You may find that using `brew bump-formula-pr` makes this very simple.

Comment (abpostelnicu) [reply Formula/llvm.rb:8]: why is this needed?

Comment (nandahkrishna) [reply Formula/llvm.rb:8]: https://docs.brew.sh/Formula-Cookbook#formulae-revisions

These are used when there are updates to a formula without a version update, or a recompile that users must upgrade to. For new versions, `revision` must be 0 (which means you can remove this line).

**Comment by abpostelnicu** [comment]
@nandahkrishna any idea why the test builds were cancelled?

**Comment by nandahkrishna** [comment]
I added the `CI-long-timeout` label a little too late, should be fine now that I've rerun it.

**Review by carlocab — APPROVED** [review]

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**